src/plugins/intel_gpu/src/graph/include/reshape_inst.h

```diff
@@ -79,6 +108,9 @@ struct typed_program_node<reshape> : public typed_program_node_base<reshape> {
         auto output_layout = this->get_output_layout();
         if (input_layout.has_dynamic_pad()) {
             auto prim = typed_desc();
+            // TODO: If outer padding exists, ouput padding propagation is not supported in the base mode
+            if (prim->mode == reshape::reshape_mode::base)
```

Comment:
Shouldn't we use propagatable_padding here too?

Reply:
It must return from base mode in case of outer padding case as discussed.